pkg/daemon/rpm-ostree.go

```diff
@@ -306,6 +316,30 @@ func (r *RpmOstreeClient) Rebase(imgURL, osImageContentDir string) (changed bool
 	return
 }
 
+// IsBootableImage determines if the image is a bootable (new container formet) image, rather than a wrapper (old container format)
+func (r *RpmOstreeClient) IsBootableImage(imgURL string) (bool, error) {
+
+	imageData, err := imageInspect(imgURL)
+	if err != nil {
+		return false, err
+	}
+
+	if isBootable, ok := imageData.Labels["ostree.bootable"]; ok {
```

Comment:
I'm OK with this temporarily as a way to distinguish old vs new format, but I also think this check is better done in the ostree stack; done in https://github.com/ostreedev/ostree-rs-ext/pull/356

Once we drop the old `machine-os-content` and hard require new format, I don't think we need to duplicate the error we'd get from rpm-ostree anyways.